packages/DataGrid/src/components/RowIndicator/RowIndicator.js

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import PropTypes from "prop-types";
+import CheckBox from "@paprika/checkbox";
+import * as sc from "./RowIndicator.styles";
+
+const propTypes = {
+  hasIndexIndicator: PropTypes.bool,
+  hasNumber: PropTypes.bool,
+  isActiveCell: PropTypes.bool,
+  isActiveRow: PropTypes.bool,
+  isChecked: PropTypes.func,
+  onCheck: PropTypes.func,
+  row: PropTypes.shape({}),
+  rowIndex: PropTypes.number,
+};
+
+const defaultProps = {
+  hasIndexIndicator: true,
+  hasNumber: false,
+  isActiveCell: false,
+  isActiveRow: false,
+  row: null,
+  rowIndex: null,
+  onCheck: () => {},
+  isChecked: () => {
+    return "unchecked";
+  },
+};
+
+function isValueNotUnchecked(value) {
+  return value !== "unchecked";
```

Comment:
Can "unchecked" be mapped to a variable instead of comparing to the primitive string?